keyboards/hfdkb/8s200/readme.md

```diff
@@ -0,0 +1,27 @@
+# 8s200
+
+![8s200](https://imgur.com/l50MY40.png)
+
+This is a customizable 65% and jis keyboard.
+
+* Keyboard Maintainer: [jonylee@hfd](https://github.com/jonylee1986)
+* Hardware Supported: 8s200
+* Hardware Availability: [Mistel AIRONE PRO Black-JIS keyboard](https://mistelkeyboard.com/products/2a25e7752bf9a3d8e66548816250028a)
```

Comment:
The entire board should be under `keyboards/mistel/airone_pro`. `hfdkb/8s200` means nothing to end-users.